pinot-common/src/main/java/org/apache/pinot/common/utils/TlsUtils.java

```diff
@@ -240,10 +230,6 @@ public static void installDefaultSSLSocketFactory(String keyStoreType, String ke
       // HttpsURLConnection
       HttpsURLConnection.setDefaultSSLSocketFactory(sc.getSocketFactory());
 
-      // Apache HTTP client 3.x
-      Protocol.registerProtocol("https",
```

Comment:
Is it safe to remove this?

Reply:
Yes, according to the conversation [here](https://github.com/apache/pinot/pull/11246). We are removing `commons-httpclient 3.1` and this set custom `sslcontext` in this way.